_includes/footer.html

```diff
@@ -6,7 +6,7 @@
     </a>
 
     <div class="edit-license">
-      <a class="-edit" target="_blank" href="{{site.github_repository_url}}/blob/gh-pages/{{page.path}}">Edit this page</a>
+      <a class="-edit" target="_blank" href="{{site.github_repository_url}}blob/master/{{page.path}}">Edit this page</a>
```

Comment:
With this, if you don't have a fork then yes it will ask you to fork (just click the green button).

<img width="813" alt="Screenshot 2020-10-30 at 03 50 07" src="https://user-images.githubusercontent.com/183140/97647344-10bf7780-1a63-11eb-94fd-2a41fdebbc42.png">

Then you will be directed straight back to file in edit mode.

```suggestion
      <a class="-edit" target="_blank" href="{{site.github_repository_url}}edit/master/{{page.path}}">Edit this page</a>
```